### DISCUSSION TITLE
Did Invoke-DbaQuery replace Invoke-DbaSqlQuery

### DISCUSSION THREAD

**Discussion by Liturgist** [dataplat/dbatools#7099] [Q&A]
I have seen many uses of `Invoke-DbaSqlQuery` on web pages. However, it does not appear to be in dbatools 1.0.136 or 1.0.137.

**Comment by wsmelton** [comment]
We renamed a number of commands around version `0.9.720`

https://github.com/sqlcollaborative/dbatools/commit/9d670506d210e8e686e15b8d9f1bd31e2dacd9b1

**Reply by wsmelton** [reply]
We actually added a function to help on that rename as well `Invoke-DbatoolsRenameHelper` at the time. (Which the function still exists in the module.)
